More CSS edits

capturethemoon opened the pull request:
I made a few adjustments to the navbar and the overall layout.

capturethemoon committed:
More CSS edits

adamayd commented:
Why are you editing the bootstrap-theme.css file?  If we decide to upgrade to bootstrap 4 down the road then we will have make all those changes again in the bootstrap 4 files.  I rather see you overwrite the bootstrap styles in the styles.css file.

capturethemoon committed:
Changes to bootstrap-theme.css

As requested, I reverted back to the original version of bootstrap-theme.css. Any future css edits will be made in the style.css file.

capturethemoon committed:
Edits to style.css file

capturethemoon commented:
Good point. I will reset bootstrap-theme.css to the original version, and add my edits to style.css instead.
